nmrfx-processor/src/test/java/org/nmrfx/processor/datasets/ProcessTest.java

```diff
@@ -32,133 +73,134 @@ public long runAndCompare(String fileName) throws IOException {
         if (fileName.contains("rs2d")) {
             refFile=Path.of(validHome, fileName, "Proc","1","data.dat").toFile();
             testFile = Path.of(tmpHome, "tst_" + fileName,"Proc","1","data.dat").toFile();
-        } else {
+        } else if (fileName.contains("ucsf")) {
+            refFile=Path.of(validHome,"ubiq_hsqc_sf.ucsf").toFile();
+            testFile = Path.of(tmpHome, "tst_ubiq_hsqc_sf.ucsf").toFile();
+        }
+        else {
             refFile=Path.of(validHome, fileName + ".nv").toFile();
             testFile = Path.of(tmpHome, "tst_" + fileName + ".nv").toFile();
         }
         // someone running the tests may not have the test files as they take up a lot of disk space
         // so, currently,  if file doesn't exist a message is printed and the test passes
-        if (!refFile.exists()) {
-            System.out.println(refFile + " doesn't exist, skipping test");
-            return 0;
-        } else {
-            executeScript(fileName);
-            long result = DatasetCompare.compare(refFile, testFile);
-            return result;
-        }
+        assumeFalse(ERR_MSG + refFile, testFilesMissing(refFile));
```

Comment:
If a user without files runs the tests, they will show up as skipped rather than passed with a message. I think this is better because its a little more obvious that user hasn't fully been able to run the tests.